cmd/talosctl/cmd/talos/rotate-ca.go

```diff
@@ -0,0 +1,112 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package talos
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/siderolabs/talos/pkg/machinery/client"
+	clientconfig "github.com/siderolabs/talos/pkg/machinery/client/config"
+	"github.com/siderolabs/talos/pkg/machinery/config"
+	"github.com/siderolabs/talos/pkg/machinery/config/encoder"
+	"github.com/siderolabs/talos/pkg/machinery/config/generate/secrets"
+	"github.com/siderolabs/talos/pkg/rotate/pki/talos"
+)
+
+var rotateCACmdFlags struct {
+	clusterState  clusterNodes
+	forceEndpoint string
+	output        string
+	withExamples  bool
+	withDocs      bool
+}
+
+// rotateCACmd represents the rotate-ca command.
+var rotateCACmd = &cobra.Command{
+	Use:   "rotate-ca",
+	Short: "Rotate cluster CAs (Talos and Kubernetes APIs).",
+	Long:  `The command starts by generating new CAs, and gracefully applying it to the cluster.`,
+	Args:  cobra.NoArgs,
```

Comment:
I think we need an arg to confirm and default would be a dry run to prevent accidentally running this

Reply:
yep, probably making dry-run a default one

Reply:
updated